client/src/modules/voucher-tools/correct/voucher-tools.correct.component.js

```diff
@@ -0,0 +1,103 @@
+angular.module('bhima.components')
+  .component('bhVoucherToolsCorrect', {
+    templateUrl : 'modules/voucher-tools/correct/voucher-tools.correct.tmpl.html',
+    controller : bhVoucherToolsCorrect,
+    bindings : {
+      source : '<',
+      onClose : '&?',
+      onSuccess : '&?',
+      showBadge : '@?',
+    },
+  });
+
+bhVoucherToolsCorrect.$inject = ['VoucherToolsService', 'TransactionService', '$translate'];
+
+function bhVoucherToolsCorrect(VoucherTools, Transactions, $translate) {
+  const $ctrl = this;
+  const VOUCHER_TOOLS_REVERSE_DESCRIPTION = 'VOUCHERS.TOOLS.REVERSE.DESCRIPTION';
+  const VOUCHER_TOOLS_CORRECTION_DESCRIPTION = 'VOUCHERS.TOOLS.CORRECT.DESCRIPTION';
```

Comment:
The entry `VOUCHERS.TOOLS.CORRECT.DESCRIPTION` is missing in the voucher.json file.